trojsten/static/js/notifications.js

```diff
@@ -0,0 +1,37 @@
+(function () {
+  const icons = {
```

Comment:
Toto by sa malo posielat z djanga, inak to bude lahko rozsynchronizovane.
Mozme prodat optional parameter do notify a pridat to aj do modelu.